admin/spec/back-end/service/pin-generation-eligibility.service.spec.js

```diff
@@ -15,38 +15,38 @@ describe('pinGenerationEligibilityService', () => {
       })
     })
     it('should return true if live pin generation is allowed', async () => {
-      const result = await pinGenerationEligibilityService.isPinGenerationAllowed('live')
+      const result = await pinGenerationEligibilityService.isPinGenerationAllowed(true)
```

Comment:
just a note in terms of readability - it would be better to have a bool called `liveCheckMode` set to true here.  currently it looks like you are saying 'yes, pin generation is allowed'